test/common/http/header_utility_test.cc

```diff
@@ -416,6 +417,7 @@ invert_match: true
   header_data.push_back(
       std::make_unique<HeaderUtility::HeaderData>(parseHeaderMatcherFromYaml(yaml)));
   EXPECT_TRUE(HeaderUtility::matchHeaders(matching_headers, header_data));
+  EXPECT_TRUE(HeaderUtility::matchHeaders(matching_empty_headers, header_data));
```

Comment:
Do we have same test but with `invert_match: false`?

Reply:
No. Added them in the latest commit. Thank for pointing it out!